docs/cloud-tutorials/application-management.md

```diff
@@ -66,18 +77,35 @@ npx dbos-cloud app versions <app-name>
 
 You can redeploy a previous version of your application by passing `--previous-version <version-id>` to the [`app deploy`](../api-reference/cloud-cli.md#npx-dbos-cloud-app-deploy) command.
 
-```
+```shell
 npx dbos-cloud app deploy --previous-version <version-id>
 ```
 
 However, this will fail if the previous and current versions have different database schemas.
+
+Also, note DBOS Cloud will keep running Transact workflows from previous versions until they complete. This means you need to be mindful of non-backward compatible database schema migrations, as they could break running workflows.
+
 For more information on schema management, see our [schema management guide](./database-management.md#database-schema-management).
 
 
 ### Deleting Applications
 
 To delete an application, run:
 
-```
+```shell
 npx dbos-cloud app delete <app-name>
 ```
+
+The application will be fully deleted once all its running workflows complete.
```

Comment:
It's deleted immediately, workflows or no

Reply:
What I mean is that the application will be in state "DELETED", and a record still present & listed in `dbos-cloud app list`, until all workflows are completed

Reply:
No, it won't, we changed that behavior (or if we didn't, we should), otherwise long-running workflows would block deletion forever